tests/Gedmo/Tree/NestedTreeRootRepositoryTest.php

```diff
@@ -174,7 +176,7 @@ public function testShouldSupportChildrenHierarchyAsHtml(): void
         // check support of the closures in rootClose
         $rootClose = static function () {return '</ul><!--rootCloseClosure-->'; };
         $childOpen = static function (&$node) {
-            return '<li class="depth'.$node['level'].'">';
+            return '<li class="depth'.($node['level'] - 1).'">';
```

Comment:
Here the same thing than https://github.com/doctrine-extensions/DoctrineExtensions/pull/2447/files#r922696534